### ISSUE TITLE
twisted.words.protocols.irc.IRCClient.dccSend should just raise NotImplementedException

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7163]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)| @cyli reported|
|-|-|
|Trac ID|trac#7163|
|Type|defect|
|Created|2014-04-15 21:53:35Z|
|Branch|https://github.com/twisted/twisted/tree/delete-dcc-send-7163|

In the process of looking at [#6994](https://github.com/twisted/twisted/issues/6994), found that twisted.words.protocols.irc.IRCClient.dccSend contains a bunch of code, a raise NotImplementedError, and a bunch of erroneous code.

It should just be deleted and a test added.

<details><summary>Searchable metadata</summary>

```
trac-id__7163 7163
type__defect defect
reporter__cyli cyli
priority__normal normal
milestone__ 
branch__branches_delete_dcc_send_7163 branches/delete-dcc-send-7163
branch_author__cyli cyli
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1397598815000000 1397598815000000
changetime__1397600315000000 1397600315000000
version__None None
owner__cyli cyli

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_1"></a>|@cyli commented|
|-|-|

(In [42298]) Branching to delete-dcc-send-7163.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Alex set owner to @cyli|
|-|-|

This looks good to me, please merge once tests pass.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set status to `closed`|
|-|-|

(In [42302]) Merge pyflakes-errors-twisted-words-6994

Author: cyli
Reviewer: Alex
Fixes: [#7163](https://github.com/twisted/twisted/issues/7163)
